python/pyspark/context.py

```diff
@@ -493,14 +505,27 @@ def setLogLevel(self, logLevel: str) -> None:
         """
         Control our logLevel. This overrides any user-defined log settings.
         Valid log levels include: ALL, DEBUG, ERROR, FATAL, INFO, OFF, TRACE, WARN
+
+        .. versionadded:: 1.4.0
+
+        Parameters
+        ----------
+        logLevel : str
+            The desired log level as a string.
+
+        Examples
+        --------
+        >>> sc.setLogLevel("WARN")
```

Comment:
```suggestion
        >>> sc.setLogLevel("WARN")  # doctest :+SKIP
```

Reply:
Let's probably add `# doctest :+SKIP` because it will affect the logging in other unittests.